src/wm_tts_service_EN.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Do we need another file ? Should we not replace wm_tts_EN.py with this one ? 
And delete all the other that we dont use anymore..